### PR TITLE
Send commerce information when user configure the plugin in production mode

### PR DESCRIPTION
**Pull request by gdespirito**
Send commerce information when user configure the plugin in production mode

### CODE REVIEW

File: src/transbank_webpay.php [hunk 1]
```diff
@@ -70,11 +74,19 @@ function __construct(&$subject, $config) {
...
                 $this->updateConfig();
+                $this->sendPluginVersion();
+                print_r($this->getAllConfig(), true);
             } else if (isset($_GET['checkTransaction'])) {
                 $this->checkTransaction();
             }
         }
     }
+    
+    public function plgVmOnBeforeUserfieldSave( $plgName , &$data, &$field )
```
Comment (saengate) [inline src/transbank_webpay.php]: this maybe shouldn't be here.

File: src/transbank_webpay.php [hunk 2]
```diff
@@ -70,11 +74,19 @@ function __construct(&$subject, $config) {
                 $this->createPdf();
             } else if (isset($_GET['updateConfig'])) {
                 $this->updateConfig();
+                $this->sendPluginVersion();
+                print_r($this->getAllConfig(), true);
```
Comment (saengate) [inline src/transbank_webpay.php]: Good work! I would never have found this location.

File: src/transbank_webpay.php [hunk 3]
```diff
@@ -70,6 +74,8 @@ function __construct(&$subject, $config) {
                 $this->createPdf();
             } else if (isset($_GET['updateConfig'])) {
                 $this->updateConfig();
+                $this->sendPluginVersion();
```
Comment (saengate) [inline src/transbank_webpay.php:77]: Estas lineas me llaman la atención, no se si son necesarias. Sobre todo por el print_r.
```suggestion
                $this->sendPluginVersion();
```

File: src/transbank_webpay.php [hunk 4]
```diff
@@ -70,6 +74,8 @@ function __construct(&$subject, $config) {
...
+                print_r($this->getAllConfig(), true);
```
Comment (saengate) [inline src/transbank_webpay.php]:
```suggestion
```

Comment (gdespirito) [inline src/transbank_webpay.php]:
```suggestion
                
```

Comment (gdespirito) [inline src/transbank_webpay.php]:
```suggestion
                
```

**Review by saengate — APPROVED** [review]

**Pull request merged**